source/templates/classic/image.svg

```diff
@@ -897,12 +897,12 @@
 
       <% if (base.metadata) { %>
         <footer>
-          <span>These metrics <%= !computed.token.scopes.includes("repo") ? "does not include all" : "includes" %> private contributions<% if ((config.timezone?.name)&&(!config.timezone?.error)) { %>, timezone <%= config.timezone.name %><% } %></span>
+          <span>These metrics <%= !computed.token.scopes.includes("repo") ? "do not include all" : "include" %> private contributions<% if ((config.timezone?.name)&&(!config.timezone?.error)) { %>, timezone <%= config.timezone.name %><% } %></span>
           <span>Last updated <%= new Date().toGMTString() %> with lowlighter/metrics@<%= meta.version %></span>
```

Comment:
Can you tell me what's the output for just using `<%= new Date() %>` ? 
For me it'll return `Sat Jan 09 2021 09:50:28 GMT+0100 (heure normale d’Europe centrale)` which is a bit long but maybe that's just because of my locale set to French

Reply:
The bit in brackets is just the full name of the timezone in your locale, so for me it would be `Sun Jan 10 2021 07:57:40 GMT+1300 (New Zealand Daylight Time)`. Note however that I didn't touch that line, it's just in red for some reason.

Reply:
Ah yes never mind, didn't notice it was syntaxic coloration